resolwe_bio/management/commands/generate_samples.py

```diff
@@ -0,0 +1,248 @@
+                                  'source': 'brain',
+                                  'cell_type': random.choice(['neuron', 'glia']),
+                                  'genotype': random.choice(['wildtype', 'CDK6-/-']),
+                                  'experiment_type': 'RNA-Seq',
+                                  'molecule': 'polyA RNA',
+                                  'optional_char': ['Cell line: DBTRG-05MG'],
+                                  'description': "Human brain study. Expression profiling."},
+                          'protocols': {
+                            'fragmentation_method': 'sonication',
+                            'growth_protocol': growth_protocol,
+                            'library_prep': library_prep}}
+        if name.startswith('Mm'):
+            annotation = {'geo': {'organism': 'Mus musculus',
+                                  'annotator': random.choice(annotator),
+                                  'source': 'brain',
+                                  'genotype': random.choice(['wildtype', 'CDK6-/-']),
+                                  'cell_type': random.choice(['neuron', 'microglia']),
+                                  'experiment_type': 'RNA-Seq',
+                                  'molecule': 'polyA RNA',
+                                  'optional_char': ['Cell line: EOC 2'],
+                                  'description': "Mouse brain study. Expression profiling."},
+                          'protocols': {
+                            'fragmentation_method': 'sonication',
+                            'growth_protocol': growth_protocol,
+                            'library_prep': library_prep}}
+        return annotation
+
+    def create_data(self, reads_name='seq_reads', annotated=False):
+        # get test data paths
+        data_path = settings.FLOW_EXECUTOR['DATA_PATH']
+        path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
+        reads = os.path.join(path, 'tests', 'files', reads_name + '.fastq.gz')
+        fastqc = os.path.join(path, 'tests', 'files', reads_name + '_fastqc.zip')
+        bam_mapping = os.path.join(path, 'tests', 'files', 'alignment_position_sorted.bam')
+        bai = os.path.join(path, 'tests', 'files', 'alignment_position_sorted.bam.bai')
+        dicty_genes = os.path.join(path, 'tests', 'files', 'dicty_genes.tab.gz')
+        human_genes = os.path.join(path, 'tests', 'files', 'human_genes.tab.gz')
+        mouse_genes = os.path.join(path, 'tests', 'files', 'mouse_genes.tab.gz')
+
+        # crate reads data object
+        d = Data.objects.create(
+            name = self.set_name(),
+            descriptor_schema = DescriptorSchema.objects.get(name='Reads annotation'),
+            descriptor = self.set_reads_descriptor(),
+            status = 'OK',
+            process = Process.objects.get(slug='import-upload-reads-fastq'),
+            contributor = User.objects.get(username='admin'),
```

Comment:
Get user by filtering `is_superuser=True` and check if any such user exists. You can check how to do this here: https://github.com/genialis/resolwe/blob/master/resolwe/flow/management/commands/register.py#L213